public/apps/configuration/panels/tenant-list/tenant-list.tsx

```diff
@@ -45,9 +45,8 @@ export function TenantList(props: TenantListProps) {
   React.useEffect(() => {
     const fetchData = async () => {
       try {
-        await setIsMultiTenancyEnabled(
-          (await getDashboardsInfo(props.coreStart.http)).multitenancy_enabled
-        );
+        const dashboardsInfo = await getDashboardsInfo(props.coreStart.http);
+        setIsMultiTenancyEnabled(dashboardsInfo?.multitenancy_enabled || false);
```

Comment:
lint? Or why was this introduced?

Reply:
for better readability. The original code was awaiting on updating the state, because it internally waited on dashboardsinfo request to complete. I have separated the two out